guide/creating-your-bot/event-handling.md

```diff
@@ -2,7 +2,7 @@
 
 Node.js uses an event-driven architecture, making it possible to execute code when a specific event occurs. The discord.js library takes full advantage of this. You can visit the <DocsLink path="class/Client" /> documentation to see the full list of events.
 
-Here's the base code we'll be using:
+If you've followed the guide up to this point, your `index.js` file will have listeners for two events; `ready` and `interactionCreate`:
```

Comment:
```suggestion
If you've followed the guide up to this point, your `index.js` file will have listeners for two events: `ready` and `interactionCreate`.
```
Additionally, this could use docs links for the events

Reply:
Not sure about docs links again on this one if I update them earlier, I was intending just to name them here. Punctuation changes are fine though.